config/templates/hpc.template

```diff
@@ -59,6 +59,7 @@ nav:
       - OpenFOAM: openFOAM.md
       - Python: python.md
       - Python virtual environments: setting_up_python_virtual_environments.md
+      - R Libraries: rlibs.md
```

Comment:
Should be "packages", not libraries; see terminology used in https://cran.r-project.org/

```suggestion
      - R packages: r-packages.md